sdk/appconfiguration/app-configuration/test/internal/node/http.spec.ts

```diff
@@ -128,6 +129,7 @@ describe("http request related tests", function () {
       client =
         createAppConfigurationClientForTests({
           syncTokens: syncTokens,
+          testCredential: new NoOpCredential(),
```

Comment:
Maybe we should add a comment here to explain why we're forcing the use of `NoOpCredential` in all test modes here?